src/main/java/com/znsio/teswiz/listener/CucumberScenarioListener.java

```diff
@@ -79,9 +80,9 @@ private void webCaseFinishedHandler(TestCaseFinished event) {
                 "$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$   TEST-CASE  -- " + scenarioName + "  ENDED   $$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$");
     }
 
-    private void webRunFinishedHandler(TestRunFinished event) {
-        LOGGER.info("webRunFinishedHandler: " + event.getResult().toString());
-        LOGGER.debug("webRunFinishedHandler: rp.launch.id: " + System.getProperty("rp.launch.id"));
+    private void testRunFinishedHandler(TestRunFinished event) {
+        LOGGER.info("testRunFinishedHandler: " + event.getResult().toString()+" ,Platform: "+Runner.getPlatform());
```

Comment:
Can we not call this method multiple times?

Reply:
updated Runner.getPlatform() to a class variable and used that instead